tests/test_validation.py

```diff
@@ -0,0 +1,169 @@
+import shutil
+from pathlib import Path
+
+import great_expectations as ge
+import pandas as pd
+import pytest
+from great_expectations.core.batch import RuntimeBatchRequest
+from great_expectations.exceptions.exceptions import CheckpointNotFoundError
+from prefect import flow
+
+from prefect_great_expectations.validation import (
+    GreatExpectationValidationError,
+    run_checkpoint_validation,
+)
+
+CURRENT_DIR = Path(__file__).parent.resolve()
+CONTEXT_ROOT_DIR = CURRENT_DIR / "great_expectations"
+DATA_PATH = CURRENT_DIR / "data"
+
+
+@pytest.fixture(autouse=True)
+def disable_usage_stats(monkeypatch):
```

Comment:
Does this get triggered without any calls to it?

Reply:
Yes, because `autouse` is set to `True`